packages/orchestrator-ui-components/src/components/WfoForms/UserInputForm.tsx

```diff
@@ -384,12 +384,13 @@ function UserInputForm({
     router,
     stepUserInput,
     validSubmit,
-    cancel,
+    cancel = () => {},
     previous = () => {},
     hasNext = false,
     hasPrev = false,
     userInput,
-    isTask,
+    isTask = false,
+    isResuming = false,
```

Comment:
Fair enough. But there is a decision to take the form functionality as is from v1 as much as possible for now. This would require a refactor that's outside the scope of that IMO.